Source/Extensions/Blazorise.DataGrid/Models/DataGridRowInfo.cs

```diff
@@ -48,7 +48,7 @@ public DataGridRowInfo( TItem item, IEnumerable<DataGridColumn<TItem>> columns )
     /// <param name="hasDetailRow">DetailRow evaluation result.</param>
     /// <param name="toggleable">If true toggles the detail row.</param>
     public void SetRowDetail( bool hasDetailRow, bool toggleable )
-        => this.hasDetailRow = ( toggleable && !this.hasDetailRow & hasDetailRow ) || ( !toggleable && hasDetailRow );
+        => this.hasDetailRow =  toggleable && !this.hasDetailRow & hasDetailRow  ||  !toggleable && hasDetailRow ;
```

Comment:
Wth? Did vs just remove parentheses here? Isn't this probably now logically diff? I'll put them back just in case... O.o